docs/design/properties_code_point_trie.md

```diff
@@ -42,7 +42,7 @@ In the binary properties case, we can reduce the scope to there being only two d
 
 The use cases, or manner of data access, inform the designs of APIs and data structures.
 For regular expression parsers (regex), we need to support a text description of a _set of code points_ sharing a property.
-In this case, returning a [`CodePointInversionList`](https://icu4x.unicode.org/doc/icu/collections/codepointinvlist/struct.CodePointInversionList.html) (a set of code points, a.k.a. [`UnicodeSet`](https://unicode-org.github.io/icu/userguide/strings/unicodeset.html) in ICU) provides the most efficient usable data.
```

Comment:
Issue: I would really like the canonical docs link to be on `icu4x.unicode.org` (or docs.rs).

Reply:
Why docs.rs? Internal documentation might point at things that are not published yet.

I'm not sure I agree that `icu4x.unicode.org` would be better. The `github.io` domain kind of signals that these are dev artifacts and it's not a polished release.

Reply:
This PR correctly contains some links to docs.rs.

Fair point about `github.io`. We don't need to make that call right now.